cypress/integration/login.js

```diff
@@ -21,11 +21,50 @@ describe('Sourcebooks login', function() {
         cy.get('.page__title').contains('Timesheets');
         cy.get('.calendar').should('be.visible');
         cy.get('.tile.form').should('be.visible');
-        cy.get('.user-info__title').contains('Demo User');
+        cy.get('.user-info__title').contains('Dominykas Poškus');
         cy.get('.main-nav').find('li').should('have.length', 2);
     })
-    
+
     it('Calendar should show todays date', function(){
         cy.get('.calendar--today').find('.calendar__date').contains((new Date()).getDate());
     })
+
+    it('Verify all user roles can log in and should see appropriate tabs', function(){

```

Comment:
It would be very nice to have dynamic name of test, like "Verify that Admin role can login and see 6 tabs" and so on.
You can do that by putting roles object before the test and then writing test name:
_"Verify that" + Roles[i].name + "..."_

Reply:
This code review remark is not fixed